src/data-service/GameData.js

```diff
@@ -0,0 +1,19 @@
+
+const GameModel = require('../model/GameDto');
+const InitialConnect = require('./SequelizeConnection');
+
+let GameDataController = {};
```

Comment:
Find better names for your files and the objects/classes you are exporting, they are named something but implement a different thing inside